models/lorenz_96_tracer_advection/matlab/RunAllTests.m

```diff
@@ -0,0 +1,201 @@
+function RunAllTests(dummy)
```

Comment:
what is this file?

Reply:
i looked in models/lorenz_96/matlab and the same file has been there since 2009, with minor updates through the years.  it apparently runs a bunch of the matlab diagnostics but i'm not sure who would find this given where it's located.

Reply:
removed matlab directory (leftover from lorenz_96) see https://github.com/NCAR/DART/tree/tracer_advection_siparcs